Makefile

```diff
@@ -46,6 +47,9 @@ ARCH := $(shell uname -m)
 
 COMMIT_HASH := $(shell ./misc/hash.sh)
 
+PYVERSION := $(shell echo ${COMMIT_HASH} | sed 's/^v//;s/-/+/;s/-/./g')
```

Comment:
yes, this is a pip (or maybe setuptools?) restriction on how version numbers can look. I'll add note to the makefile